src/epfl-toggle/index.js

```diff
@@ -16,20 +16,24 @@ const {
 const {
     PanelBody,
 	RadioControl,
+	TextControl,
 } = wp.components;
 
 const { Fragment } = wp.element;
 
 registerBlockType( 'epfl/toggle', {
 
 	title: __( 'EPFL Toggle', 'epfl'),
-	description: 'v1.0.4',
+	description: 'v1.0.5',
```

Comment:
Comme on ajoute une fonctionnalitée, je te propose de mettre à jour le champ "Minor"
```suggestion
	description: 'v1.1.0',
```